WordPress/Classes/ViewRelated/Gutenberg/Collapsable Header/CollapsableHeaderViewController.swift

```diff
@@ -49,14 +51,19 @@ class CollapsableHeaderViewController: UIViewController, NoResultsViewHost {
     @IBOutlet weak var selectedStateButtonsContainer: UIStackView!
     @IBOutlet weak var seperator: UIView!
 
+    // Flag indicating if the action button stack view (selectedStateButtonsContainer) is vertical.
+    // Used when calculating the footer height.
+    private var verticalActionButtons: Bool = false
```

Comment:
Really just an optional preference of mine; how about using `usesVerticalActionButtons`? At a glance, the current name reads like it would return an array.

Reply:
Can do!